### ISSUE TITLE
Hand Position -4 on page load is incorrect

### ISSUE THREAD

**Issue by terracoda** [phetsims/john-travoltage#282]
Position negative 4 on page load should be:
> Hand pointing at upper door, close to doorknob.

After a recent commit, it just changed to:
> Away from doorknob. (see screen shot)

On a sim reset, the position description is correct.
![screen shot 2018-06-05 at 11 59 37 am](https://user-images.githubusercontent.com/1902116/40982529-0387de86-68b8-11e8-9f34-a0a6085449b2.png)

**Comment by terracoda** [comment]
@mbarlow12, I am assigning to you as you made the last commit. Please re-assign if it should be someone else.

**Comment by terracoda** [comment]
"Away from doorknob." is a progress alert, not an actual position. I hope that makes sense ;-)

**Comment by terracoda** [comment]
@mbarlow12, I think this issue may be related to changes in the interaction in https://github.com/phetsims/john-travoltage/issues/267. It may fix itself, once you finished your work on that issue.

The starting position has to be a "position description" and not a "progress description".

I said "progress alert" in my previous comment, but I just looked at the sim and see that "progress descriptions" are technically not implemented as alerts, but rather as dynamic values on the Hand position, slider.

**Comment by mbarlow12** [comment]
Thanks for the info, @terracoda. It'll take me some time to review how @jessegreenberg mapped the alerts/descriptions to their respective sim events.

**Comment by jessegreenberg** [comment]
Let me know if I can help clarify if the mapping isn't clear, Im sure documentation could be improved.

**Comment by terracoda** [comment]
@mbarlow12, yes, the Hand position slider has 12 "regions" and within those regions exist 9 "landmarks", and then there is some logic that takes care of some dynamic "progress descriptions" like changes in direction. 

I'll post a couple of figures to help out.

**Comment by terracoda** [comment]
This may not be a perfect figure, but I think it clearly communicates Landmarks and Region Descriptions ([RD]).
![screen shot 2018-06-08 at 10 07 56 am](https://user-images.githubusercontent.com/1902116/41158560-f3d9e56c-6b03-11e8-981b-832b28aa5594.png)

**Comment by mbarlow12** [comment]
Thanks for the update, @terracoda. I had meant to ask—is this bug impeding work that you're trying to complete? If so, I can address this ASAP.

**Comment by terracoda** [comment]
@mbarlow12, thanks for asking. Nope, it is not impeding anything on my end. I just reported because I just happen to see it while working on Friction. Just want to make sure it does not slip through into a maintenance release ;-)

**Comment by terracoda** [comment]
@mbarlow12, I just

**Comment by terracoda** [comment]
Oops. as I was saying, @mbarlow12, I just went to verify the fix on master and I only hear and see the position numbers, not the actual aria-valuetext values.

Perhaps, you are still implementing that part?

**Comment by terracoda** [comment]
<img width="994" alt="screen shot 2018-06-28 at 10 04 53 pm" src="https://user-images.githubusercontent.com/1902116/42067148-53d21b3a-7b1f-11e8-9941-a55fb4f316c6.png">

And I noticed that the H1 has some strange addition, but that is likely related to changes @zepumph is working on over in https://github.com/phetsims/area-model-common/issues/149#issuecomment-398909150

**Comment by zepumph** [comment]
above is fixed, please full.

**Comment by mbarlow12** [comment]
@terracoda  the position is correct (-4); however, the implementation with AccessibleSlider changes how `aria-valuetext` is set. The new issue is recorded in https://github.com/phetsims/john-travoltage/issues/285. Closing this issue and will open others as needed.

**Comment by terracoda** [comment]
@mbarlow12, thanks for the explanation. The position number was always correct, it was the aria-valuetext that was incorrect, so that's why I was confused when the issue was closed and the text was missing.
